packages/toolkit/package.json

```diff
@@ -103,7 +103,7 @@
     "format": "prettier --write \"(src|examples)/**/*.{ts,tsx}\" \"**/*.md\"",
     "format:check": "prettier --list-different \"(src|examples)/**/*.{ts,tsx}\" \"docs/*/**.md\"",
     "lint": "eslint src examples",
-    "test": "vitest --run --typecheck",
+    "test": "vitest --typecheck --run ",
```

Comment:
Just out of curiousity, what's the difference?

Reply:
Argument ordering.  If I do `yarn test sorted_state_adapter`, I want that filename to get passed through as a modifier to `--run` so that it filters files.  If you do it with `--typecheck` last, it ignores it, and runs _all_ test files instead.

Reply:
ah I see, seems like it's not an issue anymore in the latest version of `vitest`.